db/compaction/compaction_service_job.cc

```diff
@@ -585,7 +618,16 @@ static std::unordered_map<std::string, OptionTypeInfo>
             const auto this_one = static_cast<const TableProperties*>(addr1);
             const auto that_one = static_cast<const TableProperties*>(addr2);
             return this_one->AreEqual(opts, that_one, mismatch);
-          }}}};
+          }}},
+        {"is_penultimate_level_output",
```

Comment:
Before we proliferate this name into schema, I think we should consider the best long term name for it. Based on a proposal in https://fburl.com/gdoc/4jqjm63t, I suggest `is_proximal_level_output`. Do you have an opinion @cbi42 ?

I'm OK to clean up/reconcile the rest of the "penultimate" naming in the codebase in a later PR.